alibi/confidence/__init__.py

```diff
@@ -3,9 +3,9 @@
 """
 
 from .trustscore import TrustScore
-from .model_linearity import LinearityMeasure
-from .model_linearity import linearity_measure
+from .model_linearity import LinearityMeasure, linearity_measure, _infer_feature_range
```

Comment:
Why does `_infer_feature_range` need to be added here?

Reply:
It's used in an [example](https://docs.seldon.io/projects/alibi/en/latest/examples/linearity_measure_fashion_mnist.html?highlight=_infer_feature). I'll refactor to be public though!